LICENSE.md

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2018 The Children's Hospital of Philadelphia 
```

Comment:
These copyright assertions aren't super important, so don't get caught up too much in perfecting it, but I don't think CHOP is the copyright owner of @gwaygenomics' contributions.

This should either be @gwaygenomics or University of Pennsylvania. You could always do something like:

> Copyright (C) 2018 Contributors, The Children's Hospital of Philadelphia, and the University of Pennsylvania

to cover all the possibilities

Reply:
Also I don't think this should be h1 formatted. I would make it regular text and set a title of `# BSD 3-Clause License`

Reply:
Awesome, thanks!